deploy.yml

```
@@ -742,6 +743,15 @@ resources:
       repository: ruby
       tag: '2.5.3-alpine3.9'
 
+  # S3 Resources
+  - name: wordlist-file
+    type: s3
+    source:
+      bucket: govwifi-wordlist
+      access_key_id: "((deploy-access-key-id))"
+      secret_access_key: "((deploy-secret-access-key))"
+      versioned_file: wordlist-short
```

```suggestion
      versioned_file: wordlist-short
      region_name: eu-west-2
```

Take this for the ride.